### DISCUSSION TITLE
asking about axis-video-player package to embed the camera video in react application

### DISCUSSION THREAD

**Discussion by ReemMahm0ud** [AxisCommunications/discussions#264] [ACAP]
Hello,

i was trying to stream axis camera in my react application and i used media-stream-library-js to stream it and used the authentication url before streaming it. the authentication is working fine but the stream does not work. so i searched the web page that axis created for the camera to find how they stream the camera. and found out that they are using package called (axis-video-player) so how can i get this package to help me stream the camera in my react application like axis camera page streaming??

thanks in advance,

**Comment by vivekatoffice** [comment]
Related links:
[A catalogue of React components focused on ease-of-use.](https://github.com/AxisCommunications/practical-react-components)
